prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -24,6 +26,12 @@ def main(
     for function in OMEN_FUNCTIONS:
         engine.register(function())
 
+    # This description below serves to unique identify agent entries on the LTM, and should be
+    # unique across instances (i.e. markets).
+    unique_task_description = f"microchain-agent-demo"
```

Comment:
If it's supposed to be unique across markets, shouldn't there be some market identifier in it?

Reply:
Yes, market_identifier was missing, thanks